### PR TITLE
Fixed small problem when random_state = 0

### PR DESCRIPTION
**Pull request by Yatoom**
Small fix for checking if random_state is set, and documentation fix.

### CODE REVIEW

**Comment by rhiever** [comment]
Much appreciated! Can you please PR this to the `development` branch instead?

File: tpot/base.py [hunk 1]
```diff
@@ -286,7 +286,7 @@ def fit(self, features, classes):
         features = features.astype(np.float64)
 
         # Set the seed for the GP run
-        if self.random_state:
+        if self.random_state is not None:
```
Comment (rhiever) [inline tpot/base.py]: Maybe this should be:

```python
if not (self.random_state is None):
```

Comment (Yatoom) [inline tpot/base.py]: I went to look up what the difference was. According to this thread hey compile to the same bytecode: http://stackoverflow.com/questions/2710940/python-if-x-is-not-none-or-if-not-x-is-none

Comment (rhiever) [inline tpot/base.py]: Works for me! 👍

**Pull request merged**